third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -2720,16 +2746,28 @@ fn parse_struct_decl(
                     list.push(variant)
                 }
                 consume_token(context.tokens, Tok::RBrace)?;
+                parse_postfix_abilities(context, &mut abilities)?;
```

Comment:
I suppose postfix abilities are not allowed in `enum` based on the previous discussions and the grammar above.

Reply:
It's supported now for both structs and enums. See `third_party/move/move-compiler-v2/tests/checking/positional_fields/variant_ability_decl_ok.move`

Reply:
Do you mean postfix abilities are valid in these cases?
```move
module 0x1234::post_abilities {
  enum SomeEnum {
    ...
  } has store;

  struct SomeStruct (...) has drop, key;
}
```

Reply:
Yes.